src/xdocs/config_filters.xml

```diff
@@ -632,6 +633,160 @@ public class UserService {
         <p> <a href="config.html#Checker">Checker</a> </p>
       </subsection>
     </section>
+    <section name="SuppressionXpathFilter">
```

Comment:
Comment out the documentation as we are not ready to make it public available

Reply:
Commenting or removing section gives following message: 
> AllChecksTest.testAllCheckstyleModulesHaveXdocDocumentation:425->lambda$testAllCheckstyleModulesHaveXdocDocumentation$3:429 Module SuppressionXpathFilter does not have xdoc documentation.

Reply:
Should I rename `SuppressionXpathFilter` to `SuppressionXpathFilterStub` to avoid checks for documentation and etc?

Reply:
@timurt 
What for? Do we have problems with checking of documentation?

Reply:
@rnveach 
Can we exclude the module from UT till we finally release the whole model and make it public available?

Reply:
Sorry I missed this ping before.
https://github.com/timurt/checkstyle/blob/4421/src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java#L422-L424
Just add the module to this list with it's own comment above.